src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -117,10 +173,57 @@ class pt7(Macro):
     
     def run(self, *args, **kwargs):
         pass
+    
+class pt7d1(Macro):
+    """Macro with a list of pair Motor,Float. Default value for last ParamRepeat element.
+    Usages, ex.:
+    pt7d1 [[mot1 1][mot2 3]]
+    pt7d1 [mot1 1 mot2 3]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.